diagnostic.md

```diff
@@ -36,19 +36,22 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
+The path listed above is a relative path because it tells you exactly where to go from the Home directory. It gives a direct route, avoiding online URLs that may change over time and therefore lose the information they linked to.
```

Comment:
Actually, because the path starts from the home directory, it is an absolute path.